doc/source/ray-overview/basics.rst

```diff
@@ -25,3 +25,5 @@ Ray also has a number of other community contributed libraries:
 - :doc:`../pandas_on_ray`
 - :doc:`../joblib`
 - :doc:`../multiprocessing`
+
+Check out the :ref:`rapidly growing list of open source projects <ray-oss-list>` that are known to use Ray!
```

Comment:
```suggestion
Check out the :ref:`full list of distributed libraries <ray-oss-list>` that integrate with Ray.
```

Tone.

Reply:
Actually I think we should replace this entire section (this sentence, and the bullets above), with:

```
There are also many `community libraries` that integrate with Ray, including `Modin`, `Dask`,
`MARS`, `Horovod`, `Flambe`, and others. Check out Ray's `full list of distributed libraries`.
```

Where both community and full list link to the community libraries page. We can pick perhaps the top 5 most popular libraries to link by name (not sure which makes sense here). Then, this snippet can be easily reused not only here but in the README, intro, etc.